jellyfin-core/src/main/kotlin/org/jellyfin/sdk/discovery/AddressCandidateHelper.kt

```diff
@@ -143,21 +126,14 @@ public class AddressCandidateHelper(
 	}
 
 	/**
-	 * Reorder all candidates based on priority.
+	 * Returns all unique candidates sorted by priority.
 	 *
-	 *   - HTTPS above HTTP
-	 *   - Jellyfin ports above default ports
-	 */
-	public fun prioritize() {
-		logger.debug("Prioritizing candidates")
-		candidates.sortWith(prioritizeComparator)
-	}
-
-	/**
-	 * Get all deduplicated candidate URLs as strings.
-	 * Call [prioritize] before if a sorted list is desired.
+	 * The priority is based on a few rules:
+	 * - HTTPS before HTTP
+	 * - Jellyfin ports before protocol default ports
 	 */
 	public fun getCandidates(): List<String> = candidates
+		.sortedWith(prioritizeComparator)
 		.map { it.toString() }
 		.distinct()
```

Comment:
Probably not necessary anymore with the set? At least if the URL is bijective to its string representation wrt uniqueness/equality.

Reply:
Correct, left-over from the list implementation